test/crowdsale/CapStagedCrowdsale.test.js

```diff
@@ -0,0 +1,63 @@
+const CapStagedCrowdsale = artifacts.require('CapStagedCrowdsale');
+const BigNumber = web3.BigNumber;
+const SimpleToken = artifacts.require('SimpleToken');
+require('chai')
+  .use(require('chai-as-promised'))
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+contract('CapStagedCrowdsale', function ([_, owner, wallet, investor]) {
+  const _rate = new BigNumber(1000);
+  const _value = web3.toWei('1', 'ether');//  10ether
+  const stage1Limit = web3.toWei('1', 'ether');//  10ether
```

Comment:
Let's condense all of these variables into something more readable like

```
const stages = [
  { limit: web3.toWei(1, 'ether'), rate: new BigNumber(40) },
  { limit: web3.toWei(2, 'ether'), rate: new BigNumber(30) },
  { limit: web3.toWei(3, 'ether'), rate: new BigNumber(20) },
]

const rates = stages.map((s) => s.rate)
cont limits = stages.map((s) => s.limit)
```

Reply:
I've changed variables in a way you suggested.